pkg/tsdb/elasticsearch/response_parser_test.go

```diff
@@ -322,6 +322,79 @@ func TestResponseParser(t *testing.T) {
 			So(seriesTwo.Points[1][1].Float64, ShouldEqual, 2000)
 		})
 
+		Convey("With top_metrics", func() {
```

Comment:
Please refactor this to use `testify` instead of `convey`, as used in --> https://github.com/grafana/grafana/blob/main/pkg/api/annotations_test.go for example.